source/_integrations/signal_messenger.markdown

```diff
@@ -115,6 +115,9 @@ action:
 
 ### Text message with an attachment
 
+This example assumes you have an image stored in the default `www`-folder in Home Assistant Operating System (not Docker).
```

Comment:
How is this different from Docker?

Reply:
I do not know how this is different from docker. I can only 
- confirm validity of the proposed change for Home Assistant Operating System
- confirm that the original text did not work for Home Assistant Operating System

Unfortunately I have no possibility to check applicability for docker, but as the proposed change fixes a problem for Home Assistant Operating System users, I wanted to share it.